### PR TITLE
[fix][website] exclude outdated versions of api documentation from search engine results

### PR DESCRIPTION
**Pull request by zuobiao-zhou**
This PR fixes https://github.com/apache/pulsar/issues/20390

improved version of https://github.com/apache/pulsar-site/pull/591

cc @Anonymitaet @tisonkun 

- [ ] `doc` 
- [ ] `doc-required` 
- [x] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

**Comment by zuobiao-zhou** [comment]
image
image

**Comment by Anonymitaet** [comment]
@tisonkun 
Could you please help review this PR? Thank you!

File: static/robots.txt [hunk 1]
```diff
@@ -1 +1,40 @@
...
+Disallow: /api/*/2.9.0/
+Disallow: /api/*/2.9.1/
+Disallow: /api/*/2.9.2/
+Disallow: /api/*/2.9.3/
+Disallow: /api/*/2.9.x/
+Disallow: /api/*/2.10.0/
+Disallow: /api/*/2.10.1/
+Disallow: /api/*/2.10.2/
+Disallow: /api/*/2.10.x/
+Disallow: /api/*/2.11.x/
```
Comment (tisonkun) [inline static/robots.txt:38]: IIRC 2.10 and 2.11 are still maintained. If so, we should allow these two versions.

cc @Anonymitaet

Comment (zuobiao-zhou) [reply static/robots.txt]: updated.

**Review by Anonymitaet — APPROVED** [review]

**Comment by tisonkun** [comment]
Thank you!

**Pull request merged**